Sources/SwiftGodot/SwiftGodot.docc/Info.plist

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleName</key>
+	<string>SwiftGodot</string>
+	<key>CFBundleDisplayName</key>
+	<string>SwiftGodot</string>
+	<key>CDDefaultCodeListingLanguage</key>
+	<string>swift</string>
```

Comment:
Adding this file and `CDDefaultCodeListingLanguage`. We will not longer required to add explicitly `swift` for every codeblock in the docc doc comments.